pkg/utils/etcdutil/etcdutil.go

```diff
@@ -863,10 +863,9 @@ func (lw *LoopWatcher) watch(ctx context.Context, revision int64) (nextRevision
 	}
 }
 
-func (lw *LoopWatcher) load(ctx context.Context) (nextRevision int64, err error) {
-	ctx, cancel := context.WithTimeout(ctx, DefaultRequestTimeout)
-	defer cancel()
+func (lw *LoopWatcher) load() (nextRevision int64, err error) {
```

Comment:
maybe we still need ctx to avoid load too long?

Reply:
ok, we use external ctx to control it